config.proto

```diff
@@ -50,6 +50,9 @@ message Opa {
 
     // poll period in seconds to query OPA service
     google.protobuf.Int32Value pollPeriodSeconds  = 2;
+
+    // when `true` Open Policy Agent evaluation is enabled to block request
+    google.protobuf.BoolValue enabled = 3;
```

Comment:
@davexroth you have mentioned that there will be more evaluation filters executed. Do we want to enable/disable them all or separately?

Reply:
for now let's do all.